docs/api-reference-index.md

```diff
@@ -0,0 +1,11 @@
+# API Reference
+
+## Table of Content
+
+- [`TLNetwork`](./api/classes/_tlnetwork_.tlnetwork.md)
+  - [`CurrencyNetwork`](./api/classes/_currencynetwork_.currencynetwork.md)
+  - [`User`](./api/classes/_user_.user.md)
+  - [`Trustline`](./api/classes/_trustline_.trustline.md)
+  - [`Payment`](./api/classes/_payment_.payment.md)
+  - [`Events`](./api/classes/_events_.events.md)
+- [`Utils`](<(./api/modules/_utils_.md)>)
```

Comment:
That link is broken. Not sure if this is a typo or what was the origin intention by the `<`'s.